main.py

```diff
@@ -26,11 +26,22 @@
 import threading
 
 fileblock = threading.Lock()
-glacier = boto3.client('glacier')
+#glacier = boto3.client('glacier')
 
 
+def start(args):
+    if 'region' in args:
+        if not args['region'] == "":
+            print('connecting to region: %s' % args['region'] )
+            glacier = boto3.client('glacier', args['region'])
+    else: 
+        print('connecting to your default region')
+        glacier = boto3.client('glacier')
+    return glacier
+
 def main():
     args = parse_args()
+    glacier = start(args)
```

Comment:
While we're at it, how about renaming it to `glacier_client` to clarify it is a boto3 client?